public/usage-examples/utilities/display_dialog/display_dialog-1-show-sizes-oop.cs

```diff
@@ -0,0 +1,19 @@
+﻿using SplashKitSDK;
+
+namespace Program
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Display a dialog to the user
+            SplashKit.DisplayDialog("Welcome!", "Hello, this is a simple dialog message.", SplashKit.GetSystemFont(), 20);
+
+            // Display another dialog with a different message and font size
+            SplashKit.DisplayDialog("Second Message", "This is another dialog with a different message.", SplashKit.GetSystemFont(), 25);
+
+            // Display a dialog with a different title and message
+            SplashKit.DisplayDialog("Third Message", "This is a dialog with BIG TEXT!", SplashKit.GetSystemFont(), 40);
```

Comment:
There is some sort of issue with the display_dialog function that isn't showing the last dialog box. I've tested a few things and I think it has something to do with the length of the message because when I put in a few more characters in like "This is another dialog with BIG TEXT!" it works fine.